api/routes/projects.js

```diff
@@ -122,4 +122,73 @@ router.get('/', async (req, res, next) => {
   });
 });
 
+router.get('/single', async (req, res, next) => {
+
+  const singleRes = await axios.get(
```

Comment:
Is there a reason why the URL accepts both `res` and `it` parameters?  Could it instead only accept an `it` parameter and then the code does the job of fetching only the resources relevant to that project?  (Also then you could use a classic API URL format of `/projects/{project ID}` e.g. `/projects/IT-347`)